Tests/DependencyInjection/AbstractDoctrineExtensionTest.php

```diff
@@ -997,16 +997,16 @@ public function testPrivateLazyEntityListener() : void
 
     /**
      * @expectedException \InvalidArgumentException
-     * @expectedExceptionMessageRegExp /The service ".*" must not be abstract as this entity listener is lazy-loaded/
+     * @expectedExceptionMessageRegExp /The service ".*" must not be abstract\./
      */
-    public function testAbstractLazyEntityListener() : void
+    public function testAbstractEntityListener() : void
```

Comment:
This is not a BC break, merely misleading test case. It DID throw even for non-lazy abstract listeners before.